### PR TITLE
Migrate from ember-cli-qunit to ember-qunit.

### PR DESCRIPTION
**Pull request by rwjblue**
The API's are (as of ember-qunit@3) compatible, and all of the main functionality lives in ember-qunit (ember-cli-qunit@4.1 is just a very thin shim over ember-qunit).

This modifies the application blueprint to use ember-qunit directly (and ultimately ember-cli-qunit would become deprecated).

For discussion with @ember-cli/core...

### CODE REVIEW

**Comment by rwjblue** [comment]
@ember-cli/core - Thing its time to move forward with this?

**Comment by mehulkar** [comment]
+1 let's get this party started.

**Comment by rwjblue** [comment]
Just rebased and targeted the `beta` branch (for a 3.5 release).

File: blueprints/app/files/package.json [hunk 1]
```diff
@@ -38,6 +38,7 @@
     "ember-export-application-global": "^2.0.0",
     "ember-load-initializers": "^1.1.0",
     "ember-maybe-import-regenerator": "^0.1.6",
+    "ember-qunit": "^3.4.1",
```
[the remark this answers is not in the document]
Comment (rwjblue) [reply blueprints/app/files/package.json]: Updated

**Pull request merged**